solarforecastarbiter/tests/data/report_metadata.json

```diff
@@ -24,9 +24,9 @@
             "mbe"
         ],
         "categories": [
-          "Total",
-          "Date",
-          "Hour of the day"
+          "total",
+          "date",
+          "hour"
```

Comment:
I'm surprised/concerned that the tests pass on master.

Reply:
I think mocking interferes with the actual values